### PR TITLE
CloudFront Integration v2

### PR DESCRIPTION
**Pull request by sshelomentsev**

### CODE REVIEW

File: proxy/test/utils/customer-variables/selectors.test.ts [hunk 1]
```diff
@@ -235,201 +234,217 @@ describe('customer variables selectors', () => {
...
+    //     SecretId: secretName,
+    //   }).resolves({
+    //     SecretString: variablesRecordStr
+    //   })
+
+    //   await runAssertions(req)
+    // })
   })
 
-  describe('from secrets manager and headers', () => {
```
Comment (necipallef) [inline proxy/test/utils/customer-variables/selectors.test.ts:290]: we should consider deleting these codes if they are not needed anymore. They are in comment right now

**Review by necipallef — CHANGES_REQUESTED** [review]

File: mgmt-lambda/test/handlers/handleStatus.test.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  it('check correct output', async () => {
+    lambdaMock
+      .on(GetFunctionCommand, {
+        FunctionName: options.LambdaFunctionName,
+      })
+      .resolves({
+        Configuration: {
+          FunctionName: 'fingerprint-pro-lambda-function',
+          FunctionArn: 'arn:aws:lambda:us-east-1:1234567890:function:fingerprint-pro-lambda-function',
+          Runtime: 'nodejs16.x',
```
[the remark this answers is not in the document]
Comment (necipallef) [reply mgmt-lambda/test/handlers/handleStatus.test.ts:30]: I know it doesn't affect the test result. I would be nice if we search for nodejsX in the project and only see nodejs20

**Review by TheUnderScorer — APPROVED** [review]

**Review by necipallef — APPROVED** [review]

**Review by TheUnderScorer — APPROVED** [review]

**Pull request merged**